### ISSUE TITLE
[Anon] Bug report: Imperium - Dark Angels.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#5118]
**File:** Imperium - Dark Angels.catz

**BattleScribe version:** 2.02.01

**Platform:** Android

**Dropbox:** No

**Description:** There is no grim resolve on anything, at all, inner circle is, combat squad is but no Grim Resolve

**Comment by Mad-Spy** [comment]
Duplicate of #5038, caused by https://github.com/BattleScribe/Release/issues/47. We're waiting on a fix from the app developer.
